lib/fluent/logger.rb

```diff
@@ -63,5 +63,13 @@ def self.default
     def self.default=(logger)
       @@default_logger = logger
     end
+
+    def method_missing(m, message)
+      if m.to_s.in?(%w(debug info warn error fatal))
```

Comment:
Object#in? is added by ActiveSupport: http://apidock.com/rails/Object/in%3F
But fluent-logger-ruby doesn't have ActiveSupport, so this line raises NoMethodError.
Thus you should define `debug`, `info`, `warn`, etc methods.